crypto/BRCryptoWallet.h

```diff
@@ -203,6 +222,34 @@ extern "C" {
 
     DECLARE_CRYPTO_GIVE_TAKE (BRCryptoWallet, cryptoWallet);
 
+    /// MARK: Wallet Sweeper
+
+    typedef enum {
+        CRYPTO_WALLET_SWEEPER_SUCCESS,
+        CRYPTO_WALLET_SWEEPER_INVALID_OPERATION,
+        CRYPTO_WALLET_SWEEPER_INVALID_HASH
+    } BRCryptoWalletSweeperStatus;
+
+    extern BRCryptoWalletSweeper
+    cryptoWalletSweeperCreateAsBtc (BRCryptoKey key, BRCryptoAddressScheme scheme);
+
+    extern void
+    cryptoWalletSweeperRelease (BRCryptoWalletSweeper sweeper);
+
+    extern BRCryptoKey
+    cryptoWalletSweeperGetKey (BRCryptoWalletSweeper sweeper);
+
+    extern BRCryptoAmount
+    cryptoWalletSweeperGetBalance (BRCryptoWalletSweeper sweeper,
```

Comment:
I don't like Unit being provided. I think instead of a "public" `cryptoWalletSweeperCreateAsBtc`, there should be a `cryptoWalletCreateSweeper` that will create one (and provided the correct unit).